### PR TITLE
Added functionality for forgot-password

### PR DESCRIPTION
**Pull request by vladimirsm**

### CODE REVIEW

File: src/app/component/MyAccount/MyAccount.component.js [hunk 1]
```diff
@@ -115,6 +144,12 @@ class MyAccount extends Component {
               onMouseLeave={ () => this.setState({ isHovered: false }) }
             >
                 { this.renderLoader() }
+                { notification
```
Comment (alfredsgenkins) [inline src/app/component/MyAccount/MyAccount.component.js]: Not like this. Maybe we can use the notification component?

File: src/app/component/MyAccount/MyAccount.component.js [hunk 2]
```diff
@@ -235,7 +276,7 @@ class MyAccount extends Component {
                 <article block="MyAccount" elem="Additional">
                     <section aria-labelledby="forgot-password-labe">
                         <h4 id="forgot-password-label">Forgot password?</h4>
-                        <a href="#password-reset" onClick={ () => this.changeState(STATE_FORGOT_PASSWORD) }>
+                        <a href="#forgot-password" onClick={ () => this.changeState(STATE_FORGOT_PASSWORD) }>
```
Comment (alfredsgenkins) [inline src/app/component/MyAccount/MyAccount.component.js]: We don't need this links I suppose. Maybe we should remove them?

File: src/app/store/MyAccount/MyAccount.dispatcher.js [hunk 3]
```diff
@@ -42,10 +43,12 @@ class MyAccountDispatcher extends QueryDispatcher {
      * @returns {Promise<{status: String}>} Reset password token
      * @memberof MyAccountDispatcher
      */
-    forgotPassword(options = {}) {
+    forgotPassword(options = {}, dispatch) {
         const mutation = MyAccount.getForgotPasswordMutation(options);
-        // TODO: WHEN IMPLEMENTING ALWAYS RETURN THAT EMAIL WAS SENT!!!
-        fetchMutation(mutation);
+        fetchMutation(mutation).then(
```
Comment (alfredsgenkins) [inline src/app/store/MyAccount/MyAccount.dispatcher.js]: This was done just to have an action dispatched? The BE should always respond with success... Was it really required?

File: src/app/component/MyAccount/MyAccount.component.js (hunk 1 above)
Comment (vladimirsm) [reply src/app/component/MyAccount/MyAccount.component.js]: True. Just found it this morning.

File: src/app/store/MyAccount/MyAccount.dispatcher.js (hunk 3 above)
Comment (vladimirsm) [reply src/app/store/MyAccount/MyAccount.dispatcher.js]: Well, depends... This was done to turn off the loading indicator. 
Of course, we can always instantly show that password reset was triggered, but I think it is better to show when exactly the process is finished, so the user will know that he will receive email now.

One thing I will also add an error handling here. Like if the server didn't respond at all.

File: src/app/component/MyAccount/MyAccount.component.js (hunk 2 above)
Comment (vladimirsm) [reply src/app/component/MyAccount/MyAccount.component.js]: Will remove them all

**Pull request merged**